cmd/argocd/commands/app.go

```diff
@@ -137,13 +137,15 @@ func NewApplicationCreateCommand(clientOpts *argocdclient.ClientOptions) *cobra.
   # Create a Kustomize app
   argocd app create kustomize-guestbook --repo https://github.com/argoproj/argocd-example-apps.git --path kustomize-guestbook --dest-namespace default --dest-server https://kubernetes.default.svc --kustomize-image gcr.io/heptio-images/ks-guestbook-demo:0.1
 
+  # Create a MultiSource app
+  argocd app create guestbook --file <path-to-yaml-file>
+
```

Comment:
Reading this, I will assume that just by pointing to a YAML would give me a multi source app, but I think this is not true.

Reply:
It is true given that the yaml file contains application configured with multiple sources. I will specify that in the example.